GTMAppAuth.podspec

```diff
@@ -22,24 +22,33 @@ requests with AppAuth.
                    }
 
   s.source       = { :git => "https://github.com/google/GTMAppAuth.git", :tag => s.version }
-
-  s.source_files = "Source/*.{h,m}"
+  s.prefix_header_file = false
+  s.source_files = "GTMAppAuth/Sources/*.{h,m}",
+                   "GTMAppAuth/Sources/Public/GTMAppAuth/*.h"
+  s.public_header_files = "GTMAppAuth/Sources/Public/GTMAppAuth/*.h"
   s.requires_arc = true
 
-  s.ios.source_files = "Source/GTMOAuth2KeychainCompatibility/*.{h,m}",
-                       "Source/iOS/**/*.{h,m}"
+  s.ios.source_files = "GTMAppAuth/Sources/GTMOAuth2KeychainCompatibility/*.{h,m}",
+                       "GTMAppAuth/Sources/iOS/GTMKeychain_iOS.m"
   s.ios.deployment_target = "7.0"
 
-  s.osx.source_files = "Source/GTMOAuth2KeychainCompatibility/*.{h,m}",
-                       "Source/macOS/**/*.{h,m}"
+  s.osx.source_files = "GTMAppAuth/Sources/GTMOAuth2KeychainCompatibility/*.{h,m}",
+                       "GTMAppAuth/Sources/macOS/GTMKeychain_macOS.m"
   s.osx.deployment_target = '10.11'
 
-  s.tvos.source_files = "Source/iOS/GTMKeychain_iOS.m"
+  s.tvos.source_files = "GTMAppAuth/Sources/iOS/GTMKeychain_iOS.m"
+  s.tvos.exclude_files = "GTMAppAuth/Sources/Public/GTMAppAuth/GTMOAuth2KeychainCompatibility.h"
   s.tvos.deployment_target = '9.0'
 
-  s.watchos.source_files = "Source/iOS/GTMKeychain_iOS.m"
+  s.watchos.source_files = "GTMAppAuth/Sources/iOS/GTMKeychain_iOS.m"
+  s.watchos.exclude_files = "GTMAppAuth/Sources/Public/GTMAppAuth/GTMOAuth2KeychainCompatibility.h"
   s.watchos.deployment_target = '6.0'
 
+  s.pod_target_xcconfig = {
+    'GCC_C_LANGUAGE_STANDARD' => 'c99',
```

Comment:
This is in Firebase because we want to support building on non-Apple platforms - C99 is probably not necessary here.

Reply:
I'm assuming the same would be true of the cxxLanguageStandard property in the Package.swift file?

Reply:
Yes.

Reply:
Removed.